modules/ocf/manifests/packages/docker.pp

```diff
@@ -37,4 +37,23 @@
     require => Package['docker-engine'],
     notify  => Exec['docker-socket-update'];
   }
+
+  cron {
+    'clean-old-docker-containers':
+      command => "docker ps -a --filter status=exited | grep -E '(weeks|months|years) ago' | awk '{print \$1}' | chronic xargs -r docker rm",
```

Comment:
i guess that should be `(weeks|month|year)` so that singular of the latter two get caught?

i'd be okay with anything at least a day old and exited, but this seems fine too